src/blockly/generators/propc/variables.js

```diff
@@ -444,58 +357,28 @@ Blockly.Blocks.array_set = {
         this.setInputsInline(true);
         this.setPreviousStatement(true, "Block");
         this.setNextStatement(true);
+        this.arrayList = ['list'];
         this.updateArrayMenu();
     },
-    buildArrayMenu: function (v_list) {
-        var toConn = this.getInput('NUM').connection.targetConnection;
+    buildArrayMenu: function (arrayList) {
+        var inputTargetConnection = this.getInput('NUM').connection.targetConnection;
```

Comment:
Thanks for using full variable names. Much more readable.

Reply:
:)